src/web/templates/pages/index.html

```diff
@@ -17,11 +17,16 @@
     </section>
 
     <form method="POST" enctype="multipart/form-data">
-        <label for="file-selector" class="button button-secondary" title="Select File to Upload" tabindex=0>Select
-            File…</label>
-        <input id="file-selector" type="file" name="upload" />
-        <label for="upload-file" class="button" title="Send Selected File" tabindex=0>Send Selected File</label>
-        <input id="upload-file" type="submit" class="button" value="Upload" />
+        <div id='file-selector-div'>
+            <label id='file-selector-label' for="file-selector" class="button button-secondary" title="Select File to Upload" tabindex=0>Select
+                File…</label>
+            <input id="file-selector" type="file" name="upload" />
+        </div>
+        
+        <div id='upload-botton-div'>
+            <label for="upload-file" class="button" title="Send Selected File" tabindex=0>Send Selected File</label>
+            <input id="upload-file" type="submit" class="button" value="Upload" />
+        </div>
```

Comment:
What's this change for? I can't see any references to `#file-selector-div` or `#upload-botton-div` (typo btw) anywhere, so maybe we can just revert this file?